selfservice/hook/web_hook.go

```diff
@@ -225,23 +246,39 @@ func (e *WebHook) ExecuteRegistrationPreHook(_ http.ResponseWriter, req *http.Re
 	})
 }
 
-func (e *WebHook) ExecutePostRegistrationPostPersistHook(_ http.ResponseWriter, req *http.Request, flow *registration.Flow, session *session.Session) error {
+func (e *WebHook) ExecutePostRegistrationPrePersistHook(_ http.ResponseWriter, req *http.Request, flow *registration.Flow, id *identity.Identity, ct identity.CredentialsType) error {
```

Comment:
Why this credentials/CredentialsType is needed? I don't see any changes to use that after adding it to the function. Is a usage burden somewhere in jsonet files?